backend/installfinders/launchers/steam/steam.go

```diff
@@ -99,8 +100,18 @@ func FindInstallationsSteam(steamPath string, launcher string, platform common.L
 
 			var branch common.GameBranch
 			userConfig := manifest["AppState"].(map[string]interface{})["UserConfig"].(map[string]interface{})
-			betakey, ok := userConfig["betakey"]
-			if !ok {
+
+			var betakey string
+			found := false
+			for k, v := range userConfig {
+				if strings.EqualFold(k, "BetaKey") {
+					betakey = v.(string)
+					found = true
+					break
+				}
+			}
+
+			if !found {
```

Comment:
```suggestion

			var betakey string
			for k, v := range userConfig {
				if strings.EqualFold(k, "BetaKey") {
					betakey = v.(string)
					break
				}
			}

			if betakey == "" {
```

Simplification by removing `found` per feedback from Discord user robb4